decentraland/unity-explorer
# fix: Lock mouse when forcing a camera mode via SDK

## What does this PR change?

Fixes #1272 

Locks the mouse when entering a scene with camera lock

## How to test the changes?


1. Go to 84,-4 in goerli plaza (or any scene in particular)
2. Check that every time you enter the mouse is locked

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
